tests/runfiles/linux.run

```diff
@@ -90,7 +90,7 @@ tests = ['events_001_pos', 'events_002_pos', 'zed_rc_filter', 'zed_fd_spill']
 tags = ['functional', 'events']
 
 [tests/functional/fadvise:Linux]
-tests = ['fadvise_sequential']
+tests = ['fadvise_willneed']
 tags = ['functional', 'fadvise']
```

Comment:
This while test group can simply be moved in to `common.run`, then it'll always be run for both platforms.

Reply:
Ah OK, I didn't know if we were in a 'both platforms' situation or there were more.